CMakeLists.txt

```diff
@@ -2,7 +2,6 @@
 cmake_minimum_required(VERSION 3.15 FATAL_ERROR)
 cmake_policy(SET CMP0094 NEW)
 cmake_policy(SET CMP0075 NEW)
-cmake_policy(SET CMP0144 NEW)
```

Comment:
```
if (CMAKE_VERSION VERSION_GREATER_EQUAL "3.27.0")
    cmake_policy(SET CMP0144 NEW)
endif()
```
Sorry, I saw your message this morning and never got to it. This should both silence the warning for 3.27 and not raise errors for <3.27

Reply:
GH wouldn't let me do a code suggestion on a deleted line, so I added this fix to #3090